ui/src/criteria/concept.tsx

```diff
@@ -1,9 +1,33 @@
 import Typography from "@mui/material/Typography";
 import { DataGrid, GridColDef } from "@mui/x-data-grid";
 import React, { useContext, useEffect, useMemo, useState } from "react";
-import { useParams } from "react-router-dom";
-import { EntityInstancesApiContext } from "./apiContext";
-import { DataSet } from "./dataSet";
+import { EntityInstancesApiContext } from "../apiContext";
+import { Criteria, DataSet, Group } from "../dataSet";
+
+export class ConceptCriteria extends Criteria {
+  constructor(name: string, filter: string) {
```

Comment:
typescript constructor shorthand can reduce some boilerplate here and avoid some issues, e.g. `name` is not declared

https://dev.to/satansdeer/typescript-constructor-shorthand-3ibd